manager/pkg/utils/utils.go

```diff
@@ -0,0 +1,41 @@
+package utils
```

Comment:
just a best practice, utils is a overused name.  Can we name it something else??

Reply:
Renamed to operation.